### DISCUSSION TITLE
https://starter.obytes.com/overview/

### DISCUSSION THREAD

**Discussion by giscus** [obytes/react-native-template-obytes#226] [Docs Comments]
# https://starter.obytes.com/overview/

An overview of why we built this starter, including its features, the libraries used, and more.

https://starter.obytes.com/overview/

**Comment by chunghn** [comment]
This project is extremely helpful. I do not use every part of it, it is still a great reference for a proper project. Amazing!

**Comment by ameshkin** [comment]
This looks great however, I cannot get past the splash screen on an IOS device, iPhone 11 simulator.
